app/controllers/Payment_method_controller.js

```diff
@@ -0,0 +1,235 @@
+//  === Requirements ===
+const faker = require('faker')
+
+//  === Controller ===
+module.exports = {
+
+    index: function(request)
+    {
+        console.log("Payment_method_controller.index")
```

Comment:
All the other controllers use lower case for these console.logs. Please change these to be lower case.

Reply:
done